torchao/_models/llama/eval.py

```diff
@@ -48,7 +48,7 @@ def run_evaluation(
 
     print("Loading model ...")
     t0 = time.time()
-    model = _load_model(checkpoint_path, "cpu", precision)
+    model = _load_model(checkpoint_path, device, precision)
```

Comment:
can we get rid of this change, we want to load model on cpu so the full precision model doesn't have to be stored on cuda

ideally we want to enable the flow:
model on cpu -> apply quantization -> move model to gpu -> run

if your API doesn't work with this and needs to have the weights on teh correct device before quantization, then move it over as its passed into hte quantization code. i.e.  for int4wo quant we do:

quantize(model.to(device), int4_weight_only(group_size=groupsize))

otherwise, we move the model to gpu right as we construct the TransformerEvalWrapper

Reply:
yea sure. thanks for explaining the intended workflow.